### ISSUE TITLE
(option to) save and restore state per media playback

### ISSUE THREAD

**Issue by ahjolinna** [cmdrkotori/mpc-qt#112]
it would be nice to have a "bomi-player" like list where to pick and choose what kind of settings mpc-qt should remember for each playback that you chose last time.

I like this when I used bomi (when it was alive), as I have a lot of videos with many audio/sub tracks and it nice when you re-watch them you don't need to go and find the correct "settings" everytime ..only once...this is a nice "lazy-mans" feature.

how it looks on bomi: 
![image](https://user-images.githubusercontent.com/6993072/29201250-45613dd6-7e65-11e7-8fa7-e99297f101ae.png)

**Comment by cmdrkotori** [comment]
The conundrum with this is that it involves interfacing with mpv's file-local-options while most state in the gui is global. But perhaps I can buffer the changes instead and see where we end up.
